packages/core/package.json

```diff
@@ -263,6 +263,18 @@
                     "type": "object",
                     "description": "%AWS.configuration.description.lambda.recentlyUploaded%",
                     "default": []
+                },
+                "aws.accessAnalyzer.policyChecks.referencePolicyFilePath": {
+                    "type": "string",
+                    "default": "",
+                    "description": "File path or S3 path to a control policy for Custom Policy Checks.",
+                    "scope": "window"
```

Comment:
Q: What's the difference between scope `window` & `machine-overridable`

Reply:
`machine-overridable`can only be overridden by workspace or folder settings, whereas `window` can be overriden by user, workspace, or remote settings.